tracer.go

```diff
@@ -290,6 +293,7 @@ func (t *Tracer) startSpanWithOptions(
 			ctx.samplingState = parent.samplingState
 			if parent.remote {
 				ctx.samplingState.setFinal()
+				ctx.samplingState.localRootSpan = ctx.spanID
```

Comment:
What would this do? I thought for remote, it's just final and there-fore no samplers should be called?

Reply:
for consistency of localRootSpan property the assignment makes sense, but doesn't affect anything today.